_docs/howto/configure/index.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Configure Forseti Security Features
```

Comment:
nit: In previewing this these titles run long in the left navbar. I suggest either creating our own title and not using {{page.title}} or just renaming the doc to `title: Configure Features`